codespell_lib/data/dictionary.txt

```diff
@@ -7962,6 +7962,9 @@ extacy->ecstasy
 extarnal->external
 extarnally->externally
 extatic->ecstatic
+extedn->extend
+extedned->extended
+extedns->extends
```

Comment:
Please add corrections for `extender` and `extenders`

Reply:
Done